tests/functional/ext/broad_try_clause/broad_try_clause_extension.txt

```diff
@@ -1,4 +1,4 @@
 too-many-try-statements:5:0:10:8::try clause contains 3 statements, expected at most 1:UNDEFINED
 too-many-try-statements:12:0:17:8::try clause contains 3 statements, expected at most 1:UNDEFINED
-too-many-try-statements:19:0:27:8::try clause contains 4 statements, expected at most 1:UNDEFINED
+too-many-try-statements:19:0:25:8::try clause contains 4 statements, expected at most 1:UNDEFINED
```

Comment:
This is due to the change of no longer adding the `finally:` child to the end line number calculation of a `TryExcept` node.